tendrl/commons/utils/monitoring_utils.py

```diff
@@ -26,3 +26,28 @@ def update_dashboard(res_name, res_type, integration_id, action):
     ).save()
 
     return _job_id
+
+
+def update_graphite(res_name, res_type, integration_id, action):
```

Comment:
delete_resource_from_graphite instead of update_graphite

Reply:
Done